src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -1,25 +1,39 @@
 import React from 'react';
 import styled from 'styled-components';
+import { Helmet } from 'react-helmet';
 import { shape, string } from 'prop-types';
 import { GridItemConstrainedMedium } from '#lib/styledGrid';
 
+const AmpHead = () => (
+  <Helmet>
+    <script
+      async
+      custom-element="amp-iframe"
+      src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"
+    />
```

Comment:
I was of the thought we usually added these [here](https://github.com/bbc/simorgh/blob/latest/src/server/Document/component.jsx#L68-L71), will it make any difference wherever we add these ?

Reply:
Those are generic AMP scripts, those will be included on every AMP page. Things like amp-iframe are specific to the components being rendered so we don't want to add it on every page.

Reply:
Thanks Chris! That makes sense